### ISSUE TITLE
Notes display two Action Bars because two Action Bars are installed...?

### ISSUE THREAD

**Issue by WritingSolutions2020** [standardnotes/forum#10]
This has happened a few times. I open the Windows Standard Notes client (W7 64-bit) at which point Standard Notes displays two Action bars.

![standardnotesdoubleactionbar](https://user-images.githubusercontent.com/3333327/28193714-ac36fdf8-67f3-11e7-9653-30fd5dbc499f.png)

This has happened three or four times so far. I've learned to simply click the **X** on one of the instances to get rid of it. At which point things work well. If I close and start the W7 client the problem may or may not occur. Never can tell.

I recall this also occasionally happening with the Web (Ffox 53) client. But I don't have an instance to document right now. If I get one I'll take a screenshot and add it to this thread...

Not a big deal, but just a U.I. annoyance...

**Comment by WritingSolutions2020** [comment]
Oh look, the two instances of the Action Bar report two very different word, character, and paragraph counts. A clue, perhaps...?

**Comment by WritingSolutions2020** [comment]
A bit of additional information.

When I started the Windows client this morning, I noticed that two Notes' Action Bar displayed the same information. One of the Notes was the one shown in the screenshot I added to my original post.

1. I tried Reload-ing (twice in fact) to see if that would refresh the information. It did not.

2. I then clicked **Menu** > **Restore Disabled Components**. This again put two Action Bars at the bottom of the Note: One for the adjoining Note, one for the current Note. So this time I checked the title in the Action Bar and clicked **X** on the Action Bar for the other note.

After I did this, when I clicked each Note the Action Bar displayed the correct title and information. I then closed the W7 client and reopened it. This time each Note's Action Bar displayed the correct information.

In other words it seemed that somehow the Action Bar got, if you will, two pointers to two different Notes(?)

Cheers & hope this helps...

**Comment by WritingSolutions2020** [comment]
Even more information.

1. I'm also able to get two Action Bars in the Web client (Ffox 53). I need only to go to any note and click **Menu** > **Restore Disabled Components** to add a second instance *that points to the same note*.

2. The second Action Bar always points to one particular note. SO I deleted that note and recreated it in a new Note. The behavior remains the same: If in ANY note I click **Menu** > **Restore Disabled Components** I receive a second Action Bar that points to the same note.

In other words in both the W7 desktop client AND the Web client, Standard Notes has (what I'll refer to as) a pointer to information about a note that ever other note is somehow being given in the form of a second Action Bar.

Cheers & hope this helps...

**Comment by WritingSolutions2020** [comment]
More information.

Per the following screenshot, the Web client adds the same second Action Bar to a new Note:

![2017-07-14_standardnotestwoactionbars](https://user-images.githubusercontent.com/3333327/28222334-eb3e13dc-687b-11e7-873b-4f8709a359eb.png)

So it seems to me that there's an Action Bar object that has become permanently embedded in my 
Standard Notes account that has gotten permanently attached to Notes as follows:

* New notes are assigned an extra Action Bar that contains information about one particular note.
* After clicking **X** in the extra Action Bar, all Notes' receive a  **Menu** > **Restore Disabled Components* option.

As stated, I tried deleting and recreating the Note referred to by the second Action Bar. That didn't fix the problem...

**Comment by WritingSolutions2020** [comment]
I have finally figured out that my problem is probably arising because I somehow have two Action Bars installed

![2017-07-14_standardnotestwoactionbars](https://user-images.githubusercontent.com/3333327/28251150-2c245f58-6a2c-11e7-9514-96ccab5874e7.png)

I never intended to do that. More to the point, I can disable one of the two instances -- but how do I delete one of them?

**Comment by mobitar** [comment]
Hmm interesting. You can delete one of them by clicking on it in that menu to expand it, then click delete.
